app/controllers/recipes_controller.rb

```diff
@@ -1,7 +1,44 @@
 class RecipesController < ApplicationController
-  def index; end
+  before_action :set_user, expect: [:update]
+  def index
+    # @user=current_user
+    @recipes = @user.recipes
+  end
+
+  def new
+    # recipes/new
+    @recipe = Recipe.new
+  end
+
+  def create
+    # creates a recipe
+    # @user = current_user
```

Comment:
Kindly I suggest you to remove this comment since you are not using the @user instance variable anymore